src/Core/Config/Capability/IManageConfigValues.php

```diff
@@ -58,6 +58,16 @@ public function reload();
 	 */
 	public function get(string $cat, string $key = null, $default_value = null);
 
+	/**
+	 * Returns true, if the current config cannot be changed
+	 *
+	 * @param string $cat The category of the configuration value
+	 * @param string $key The configuration key to query
+	 *
+	 * @return bool true, if set is disabled
+	 */
+	public function isSetDisabled(string $cat, string $key): bool;
```

Comment:
Let's use a positive case instead, this will reduce the amount of boolean negation we have to do with `!`.
```suggestion
	public function isWritable(string $cat, string $key): bool;
```